src/briefcase/console.py

```diff
@@ -520,7 +523,7 @@ def _build_log(self, command) -> str:
             #     directory (e.g. /usr/local).
             # sys.prefix is updated (from base_prefix's value) to the virtual env's
             #     site-specific directory.
-            f"Virtual env:     {hasattr(sys, 'real_prefix') or sys.base_prefix != sys.prefix}\n"
+            f"Virtual env:     {hasattr(sys, 'real_prefix') or sys.base_prefix != sys.prefix}\n"  # noqa: E501 (line too long)
```

Comment:
Older versions of virtualenv are obsolete now, so I think all references to `real_prefix` can be removed.